lists/es/es-nif.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": {
+    "en": "Tax Identification Number",
+    "local": "Número de Identificación Fiscal"
+  },
+  "url": "http://www.agenciatributaria.es/AEAT.internet/en_gb/Inicio/La_Agencia_Tributaria/Campanas/Censos__NIF_y_domicilio_fiscal/Empresas_y_profesionales__Declaracion_censal__Modelos_036_y_037/Informacion/NIF_de_personas_juridicas_y_entidades.shtml",
+  "description": {
+    "en": "The Tax Identification Number, Número de Identificación Fiscal (NIF) is a unique number given to every legal persons and organisations (companies, associations, civil societies, public institutions etc.) required to pay taxes. The NIF is allocated by the Tax Agency [1]. There is no publicly available site to verify the ownership and validity of the number. Prior to 2008 the NIF number was called CIF, these two numbers are now merged, however, some still refer to the number as CIF.\n\n[1]: https://www.agenciatributaria.es/AEAT.internet/en_gb/Inicio/La_Agencia_Tributaria/Campanas/Censos__NIF_y_domicilio_fiscal/Empresas_y_profesionales__Declaracion_censal__Modelos_036_y_037/Informacion/NIF_de_personas_juridicas_y_entidades.shtml"
```

Comment:
A couple of minor edits:

`"The Tax Identification Number, Número de Identificación Fiscal (NIF), is a unique number given to every legal person and organisation (companies, associations, civil societies, public institutions etc.) required to pay taxes. The NIF is allocated by the Tax Agency [1]. There is no publicly available site to verify the ownership and validity of the number. Prior to 2008 the NIF number was called CIF. These two numbers are now merged; however, some still refer to the number as CIF.\n\n[1]: https://www.agenciatributaria.es/AEAT.internet/en_gb/Inicio/La_Agencia_Tributaria/Campanas/Censos__NIF_y_domicilio_fiscal/Empresas_y_profesionales__Declaracion_censal__Modelos_036_y_037/Informacion/NIF_de_personas_juridicas_y_entidades.shtml"`